openpype/hosts/blender/hooks/pre_pyside_install.py

```diff
@@ -91,7 +97,12 @@ def inner_execute(self):
         self.launch_context.env["PYTHONPATH"] = os.pathsep.join(python_paths)
 
         # Get blender's python executable
-        python_executable = os.path.join(pythond_dir, "bin", "python.exe")
+        python_bin = os.path.join(python_dir, "bin")
+        if platform == "windows":
+            python_executable = os.path.join(python_bin, "python.exe")
+        else:
+            python_executable = os.path.join(python_bin, python_version)
```

Comment:
Most of linux distributions have executable with 'python3.<0-9>m'
```suggestion
            python_executable = os.path.join(python_bin, python_version)
            # Check for python with enabled 'pymalloc'
            if not os.path.exists(python_executable):
                python_executable += "m"
```

Reply:
Hi ! Thanks for the suggestion, indeed, I noticed and corrected this but I missed my last checkout -b and my last commit appear in a duplicate branch ^^' 

https://github.com/kaamaurice/OpenPype/commit/ad3ad81ab5da36aa8ee7a62edc7333f210250fcd